docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

```diff
@@ -255,6 +255,44 @@ def sql_table(
 
 `write_disposition`: Can be "merge", "replace", or "append".
 
+## Incremental Configuration
+Efficient data management often requires loading only new or updated data from your SQL databases, rather than reprocessing the entire dataset. This is where incremental loading comes into play.
+
+Incremental loading uses a cursor column (e.g., timestamp or auto-incrementing ID) to load only data newer than a specified initial value, enhancing efficiency by reducing processing time and resource use.
+
+
+### Configuring Incremental Loading
+**Choose a Cursor Column**: Identify a column in your SQL table that can serve as a reliable indicator of new or updated rows. Common choices include timestamp columns or auto-incrementing IDs.
+
+**Set an Initial Value**: Choose a starting value for the cursor to begin loading data. This could be a specific timestamp or ID from which you wish to start loading data.
+
+**Apply Incremental Configuration**: Enable incremental loading with the incremental argument in your configuration.
+
+**Deduplication**: When using incremental loading, the system automatically handles the deduplication of rows based on the primary key (if available) or row hash for tables without a primary key.
+
+#### Incremental Loading Example
```

Comment:
Here there is only example for `sql_table`, but I think we also may transfer to this section example with `sql_database` from `Create your own pipeline` section p.7 and p.8 below on this page. Thus, we can remove p.7 and p.8 from `Create your own pipeline` section.